testsuite/cluster-test/src/instance.rs

```diff
@@ -167,6 +168,22 @@ impl Instance {
         &self.peer_name
     }
 
+    pub fn peer_id(&self) -> Result<String> {
```

Comment:
peer_id might be confusing here(peer_id for full node is `fn-x-y`), maybe `validator_index` is better?

Reply:
Done